packages/spacecat-shared-gpt-client/README.md

```diff
@@ -68,6 +71,38 @@ async function fetchInsights(prompt) {
 fetchInsights('How can we improve customer satisfaction?');
 ```
 
+#### Via Chat Completions endpoint
+
+```javascript
+async function fetchInsights(prompt) {
+  try {
+    const client = FirefallClient.createFrom({
+      env: {
+        FIREFALL_API_ENDPOINT: 'https://api.firefall.example.com',
+        FIREFALL_API_KEY: 'yourApiKey',
+        FIREFALL_API_CAPABILITY_NAME: 'yourCapabilityName',
+        IMS_HOST: 'ims.example.com',
+        IMS_CLIENT_ID: 'yourClientId',
+        IMS_CLIENT_CODE: 'yourClientCode',
+        IMS_CLIENT_SECRET: 'yourClientSecret',
+      },
+      log: console,
+    });
+    const options = {
+      imageUrls: ['data:image/png;base64,iVBORw0KGgoAAAA...=']
+    };
+
+    // // Internally, use Firefall's .../chat/completions endpoint.
+    const insights = await client.fetchChatCompletion(prompt, { options });
+    console.log('Insights:', insights);
```

Comment:
Change to be something other than insights in ordeer to differentiate from capabilities endpoint?